build/config/PublishData.json

```diff
@@ -50,6 +43,13 @@
             "nuget": [ "https://api.nuget.org/v3/index.json" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.3" ]
+        },
+        "dev15.6": {
+            "nugetKind": "Release",
+            "version": "2.3.*",
```

Comment:
This should be 2.7.*.

Reply:
or 2.8 ?

---
In reply to: [160266258](https://github.com/dotnet/roslyn/pull/24109#discussion_r160266258) [](ancestors = 160266258)